### ISSUE TITLE
Enhance to test addons that come with blender 

### ISSUE THREAD

**Issue by douglaskastle** [douglaskastle/blender-addon-tester#24]
Right now if you try to test an addon that comes with blender there appears to be a race condition on which addon to use. Based on the age of the file on the file system, so the chances of the old addon winning is actually greater.

Need to move existing addon out of the way for testing and return once testing is complete. I would rather not smash delete the old one.

**Comment by douglaskastle** [comment]
Decided to delete the addon in the blender release. I have it working for Ubuntu and Windows. I am stuck with macosx, partly because I don't know the file path to the scripts directory under macosx releases.

It's a bit messy, the addon needs to be deleted before calling blender. Removing it after is effectively deleting something that it is using.

**Comment by douglaskastle** [comment]
Implemented and merged into master
